bin/build-dev

```diff
@@ -8,6 +8,11 @@ export AWS_DEFAULT_REGION=us-west-2
 REGION=us-west-2
 
 aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/t1q6b4h2
+
 docker build -t civiform-dev --cache-from public.ecr.aws/t1q6b4h2/civiform-dev:latest --build-arg BUILDKIT_INLINE_CACHE=1 .
 docker tag civiform-dev:latest public.ecr.aws/t1q6b4h2/civiform-dev:latest
 docker push public.ecr.aws/t1q6b4h2/civiform-dev:latest
+
```

Comment:
Update the comments here and/or at the top of the doc to include the new action.

(huh I can't leave this on the comment at the top of the doc when I expand to see the whole file.